ksqldb-api-reactive-streams-tck/src/test/java/io/confluent/ksql/api/tck/BlockingQueryPublisherVerificationTest.java

```diff
@@ -69,10 +72,14 @@ private GenericRow generateRow(long num) {
 
   private static class TestQueryHandle implements PushQueryHandle {
 
-    private final long elements;
+    private final TransientQueryQueue queue;
 
     public TestQueryHandle(final long elements) {
-      this.elements = elements;
+      final OptionalInt limit = elements == 0 || elements == Long.MAX_VALUE
```

Comment:
did we intentionally change the condition to include `elements == 0`?

Reply:
Yes. Tests fail otherwise as the tck framework uses 0 as well.